.github/workflows/scriptTests.yml

```diff
@@ -0,0 +1,37 @@
+
+name: Scripts Tests
+on:
+  push:
+# Triggers the workflow on labeled PRs only.
+  pull_request_target:
+    types: [labeled]
+# Ensures that only the latest commit is running for each PR at a time.
+# Ignores this rule for push events.
+concurrency:
+  group: ${{ github.event.pull_request.number || github.sha }}script
+  cancel-in-progress: true
+jobs:
+  Scripts-tests:
+    if: contains(github.event.pull_request.labels.*.name, 'safe to test') || github.event_name == 'push'
```

Comment:
This workflow does not use credentials and therefore "safe to test" is not needed
```suggestion
```